clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -546,6 +546,12 @@ class CIRGenBuilderTy : public CIRBaseBuilderTy {
         loc, t, isSigned ? intVal.getSExtValue() : intVal.getZExtValue());
   }
 
+  mlir::cir::ConstantOp getConstInt(mlir::Location loc, llvm::APInt intVal) {
+    auto width = intVal.getBitWidth();
+    mlir::cir::IntType t = getUIntNTy(width);
+    return getConstInt(loc, t, intVal.getZExtValue());
```

Comment:
Leave a declaration and move the implementation to clang/lib/CIR/CodeGen/CIRGenBuilder.cpp